Tasks/MavenV2/maventask.ts

```diff
@@ -14,6 +14,7 @@ import {FindbugsTool} from 'codeanalysis-common/Common/FindbugsTool';
 import javacommons = require('java-common/java-common');
 import util = require('./mavenutil');
 
+const TESTRUN_SYSTEM = "VSTS - maven"; 
```

Comment:
'VSTS'?  Are we sure we don't want to update the names?

Reply:
I don't know what these are used for, we can update them all in a separate PR. I'll talk to @madhurig

Reply:
These are used for Kusto telemetry, we shouldn't rename them since it will break reports.